README.md

```diff
@@ -1,7 +1,19 @@
 kagekiri [![build status](https://circleci.com/gh/salesforce/kagekiri.svg?style=svg)](https://circleci.com/gh/salesforce/kagekiri)
 ====
 
-Shadow DOM-piercing `querySelector()` / `querySelectorAll()` implementation.
+Shadow DOM-piercing query APIs
+
+| API                      |     Interface     | Support |
+|--------------------------|-------------------|:------:|
+| `querySelector`          | Element, Document |    ✅    |
+| `querySelectorAll`       | Element, Document |    ✅    |
+| `getElementsByClassName` | Element, Document |    ✅    |
+| `getElementsByTagName`   | Element, Document |    🔜    |
+| `getElementsByTagNameNS` | Element, Document |    🔜    |
+| `getElementById`         |      Document     |    🔜    |
+| `getElementsByName`      |      Document     |    🔜    |
+| `matches`                |      Element      |    🔜    |
+
```

Comment:
I thought it would be weird to randomly add another API to the existing list without mentioning all the other ones. Once I finish all of the APIs we can remove the support column or format it differently. Thoughts?

Reply:
Makes sense to me.